Feauture detalles sitio

renatowilliner opened the pull request:
Se agrego boton de volver donde lo considere
Se quito la imagen harcodeade de los detalles
Se quito el valor True mostrado en la vista de detalles
Se quito descripcion y numero de pista del model
Se asigno un tamanio maximo a la imagen del complejo
Se agrego una notifiacion de confirmacion de las feautrue mas bonita
Se hizo que turnos en busca y por unirse sean una misma lista
Se dejo mas prolijo la vista de aceptar los turnos

renatowilliner committed:
Detalles sitio

Se agrego el boton de volver en lugares que lo crei necesario
Se unifico la lista de turnos para unirse o para reservar
Se modifico la vista para aceptar la gente que quiere reservar turnos
Se elimino la imagen harcodeada
Se elimino datos que no usabamos de los complejos
Se muestra si/no si la propiedad es bool

renatowilliner committed:
Notificaciones

Eliminado de campo cantidad_pistas
Implementacion de notificaciones

facundoschillino commented:
Me parece que podriamos dejar los campos numero de pistas y descripcion y solamente sacarlo de la descripcion o detalle de los complejos. Tambien podriamos agregar  mas campos al form que usan los complejos para terminar el registro.
Lo otro todo de 10

facundoschillino commented on padel/accounts/models.py:
Aca le dejaria el campo y lo añadiria al form que tienen los complejos para terminar de completar los datos.

Inwinkelried replied:
estoy de acuerdo, es un dato útil

renatowilliner replied:
Voy a dejar el campo 'cantidad_pistas' de nuevo

facundoschillino commented on padel/templates/detalle_complejo.html:
Acá dejaría el detalle completo del complejo cuando accedo a los detalles así tengo todo lo del complejo junto

Inwinkelried replied:
Si, en el detalle es donde tengo que ver toda la informacion del complejo. Direccion y numero de pistas tienen que estar.

renatowilliner replied:
Deje en el detalle direccion y numero de pistas

facundoschillino commented on padel/templates/mostrar_turnos_proximos.html:
Willi no entendi lo que me pusiste en el excel sobre esto

renatowilliner replied:
Que no pude fusionar las listas de turnos para reservar y tunos para unirse, son 2 listas distintas, primero se muestra una y dps la otra

facundoschillino commented on padel/templates/visualizar_mis_complejos.html:
Viendo esto me doy cuenta que tal vez sea buena decisión quitar la descripción de los complejos. Si dejaría el numero de pistas (Se me ocurre una feature para hacer con eso y los turnos)

Inwinkelried replied:
Volemos la descripcion, fijarse de borrarla en todos lados menos en el model, por si queremos usarla para algo mas adelante

Inwinkelried commented on padel/templates/registration/registro_complejos.html:
```diff
@@ -1,86 +1,47 @@
+          {{ form.direccion }}
+        </div>
+        <div class="form-group">
+          <label for="rango_horario">Rango horario de los turnos(no anda):</label>
+          
+        </div>
+        <div class="form-group">
+          <label for="duracion turnos">duracion turnos (no anda):</label>
```
estas dos cositas tienen que volar a mi parecer

Inwinkelried reviewed (changes requested):
Revisar comentarios y corregir para poder mergear. Muy buenas las nuevas funcionalidades de notificaciones

renatowilliner committed:
Merge branch 'Develop' into feauture_detalles_sitio

renatowilliner committed:
Correciones solicitadas

Estilo agregado para la busqueda
Direccion y numero de pista agregado en el detalle
Eliminacion de texto rango horario
Busqueda agregada en otro contenedor

renatowilliner committed:
Resolucion de conflictos

Resolucion

Inwinkelried commented on padel/templates/detalle_complejo.html:
```diff
@@ -6,14 +6,15 @@
         <ul class="list-group list-group-flush">
-          <li class="list-group-item"> <strong>Dirección:</strong> {{ complejo.direccion }}</li>
+            <li class="list-group-item"> <strong>Dirección:</strong> {{ complejo.direccion }}</li>
             <li class="list-group-item"> <strong>Numero de Pistas:</strong> {{ complejo.numero_de_pistas }}</li> 
```
aca usas complejo.numero_de_pistas pero ese atributo no existe. (es cantidad_pistas)
Sin embargo, cantidad_pistas lo borraste de models, asi que no va a traer nada. 

Si queres mostrar el numero de pistas, vas a tener que volver a agregarlo en el modelo de complejo y hacer una nueva migracion

renatowilliner committed:
Cantidad pistas
